### ISSUE TITLE
No Efficiency in CSV-Output & Fix

### ISSUE THREAD

**Issue by SBFspot** [SBFspot/SBFspot#104]
Hi,
there is a bug in V3.0.3 regarding calculation of Efficieny for CSV-Output, it's always 0. Reason is wrong variable name inside  CSVexport.cpp.

Instead of using calPacTot it should be TotalPac:
float calEfficiency = inverters[inv]->calPdcTot == 0 ? 0 : (float)calPacTot / (float)inverters[inv]->calPdcTot * 100.0;

it should be:
float calEfficiency = inverters[inv]->TotalPac == 0 ? 0 : (float)inverters[inv]->TotalPac / (float)inverters[inv]->calPdcTot * 100.0;

By.

#### This work item was migrated from CodePlex
CodePlex work item ID: '99'
Vote count: '2'

**Comment by SBFspot** [comment]
[joram@15/04/2015]
should it not be
```
float calEfficiency = inverters[inv]->calPdcTot == 0 ? 0 : (float)inverters[inv]->TotalPac / (float)inverters[inv]->calPdcTot * 100.0;
```
where you check that inverters[inv]->calPdcTot is not zero to avoid devision by zero
if TotalPac is zero, it does not matter with what you devide (as long as it is not zero)

**Comment by SBFspot** [comment]
**Issue closed by _SBF_ with comment**
Fixed in 3.2

**Reason closed**
Fixed
